src/vs/base/common/htmlElement.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Unsure about this. There is already the [`dom.ts#$`](https://github.com/microsoft/vscode/blob/b7f7c2e4f00c549eb7783edcb04937c71dc1ad0b/src/vs/base/browser/dom.ts#L1070)-util which has much overlap. We should use (and maybe extend) that instead of having a similar but different util.